test/integration/css/test/css-modules.test.js

```diff
@@ -145,24 +146,25 @@ describe('should handle unresolved files gracefully', () => {
       })
 
       it('should have correct file references in CSS output', async () => {
-        const cssFiles = await readdir(join(workDir, '.next/static/css'))
+        const cssFolder = join(workDir, '.next', 'static')
+        const cssFiles = nodeFs
+          .readdirSync(cssFolder, {
+            recursive: true,
+            encoding: 'utf8',
+          })
+          .filter((f) => f.endsWith('.css'))
 
-        for (const file of cssFiles) {
-          if (file.endsWith('.css.map')) continue
+        expect(cssFiles).not.toBeEmpty()
 
-          const content = await readFile(
-            join(workDir, '.next/static/css', file),
-            'utf8'
-          )
-          console.log(file, content)
+        for (const file of cssFiles) {
+          const content = await readFile(join(cssFolder, file), 'utf8')
 
           // if it is the combined global CSS file there are double the expected
           // results
           const howMany =
             content.includes('p{') || content.includes('p,') ? 2 : 1
 
           expect(content.match(/\(\/vercel\.svg/g).length).toBe(howMany)
-          // expect(content.match(/\(vercel\.svg/g).length).toBe(howMany)
           expect(content.match(/\(\/_next\/static\/media/g).length).toBe(1)
```

Comment:
This is 2 with Turbopack, this test needs to be investigated again later separately.